docs/components/sidecar.md

```diff
@@ -27,7 +27,7 @@ Prometheus servers connected to the Thanos cluster via the sidecar are subject t
 If you choose to use the sidecar to also upload data to object storage:
 
 * Must specify object storage (`--objstore.*` flags)
-* It only uploads uncompacted Prometheus blocks. For compacted blocks, see [Upload compacted blocks](./sidecar.md/#upload-compacted-blocks).
```

Comment:
Do links like this work on website? 🤔

Reply:
It works if it is in the form `./sidecar.md/#upload-compacted-blocks`. This can be fixed by transform. I was planning to add it after [this](https://github.com/bwplotka/mdox/pull/47) gets merged!